schema/server.proto

```diff
@@ -102,8 +104,12 @@ message Server {
         Endpoint.Type            endpoint_type    = 7;
         PackageRef               ingress_provider = 8;
         repeated DomainSpec      ingress_domain   = 10;  // Only has a meaning if this is an ingress.
-
-        reserved 2;
+        repeated IngressAnnotation ingress_annotation = 11;
```

Comment:
Would it make sense to use "ServiceAnnotations" here as well?

Reply:
Yes I was unsure about this but certainly we can